packages/elements/src/item/index.ts

```diff
@@ -204,6 +219,44 @@ export class Item extends ControlElement {
     }
   }
 
+
+  /**
+   * Get Item content
+   * @returns return item content from slot or label and sub-label
+   */
+  private getItemContent (): string {
+    if (this.isSlotEmpty) {
+      let text = '';
+      if (this.label) {
+        text += this.label;
+      }
+      if (this.subLabel) {
+        text += ` (${this.subLabel})`;
+      }
+      return text;
+    }
+    else {
+      return this.textContent || '';
+    }
+  }
+
+  /**
+   * Get element overflown
+   * @param element Target element
+   * @returns {boolean} Return true if element is overflown.
```

Comment:
We do not put types into comments. Types are only relevant for pure JavaScript code. IDE can resolve TypeScript types automatically.

Reply:
Updated.